src/components/Editable.js

```diff
@@ -248,6 +250,12 @@ const Editable = ({ disabled, isEditing, thoughtsRanked, contextChain, cursorOff
     dispatch(setEditingValue(newValue))
 
     if (newValue === oldValue) {
+
+      // if new value starts with space then trigger indent
+      if (isDocumentEditable() && !readonly && !immovable && state.cursor && e.target.value.startsWith(' ')) {
```

Comment:
`isDocumentEditable()` is unnecessary since the `Editable` is [disabled in Thought.js](https://github.com/cybersemics/em/blob/dev/src/components/Thought.js#L351).

Reply:
Sure. I will remove it.